cmd/gerrit/gerrit.go

```diff
@@ -808,6 +824,24 @@ func (j *DSGerrit) EnrichItem(ctx *shared.Ctx, item map[string]interface{}) (ric
 		summary = summary[:shared.KeywordMaxlength]
 	}
 	rich["summary"] = summary
+	commitMessage, ok := review["commitMessage"].(string)
+	commitBody := ""
+	if ok {
+		ary := strings.Split(commitMessage, "\n")
+		lines := []string{}
+		for _, line := range ary {
+			line = strings.TrimSpace(line)
+			if line == "" {
+				continue
+			}
+			lines = append(lines, line)
+		}
+		if len(lines) > 1 {
+			commitBody = strings.Join(lines[1:], "\n")
```

Comment:
Although, this is looking fine.
Any specific reason for this logic to prepare the `commitBody`?